backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -135,6 +134,7 @@ open class SubmissionController(
             ),
         ],
     )
+    @ApiResponse(responseCode = "304", description = "Not Modified")
```

Comment:
lets maybe add a test for this - we can basically just copy the test we had for get-released-data?

Reply:
done